third_party/islet/include/islet.h

```diff
@@ -0,0 +1,69 @@
+// Copyright (c) 2023 Samsung Electronics Co., Ltd. All Rights Reserved.
+
+#pragma once
+
+#include <cstdarg>
+#include <cstdint>
+#include <cstdlib>
+#include <ostream>
+#include <new>
+
+constexpr static const int ISLET_E_SUCCESS = 0;
+
+constexpr static const int ISLET_E_FAILURE = -1;
+
+constexpr static const int ISLET_E_INPUT = -2;
+
+constexpr static const int ISLET_E_WRONG_REPORT = -3;
+
+constexpr static const int ISLET_E_WRONG_CLAIMS = -4;
+
+constexpr static const int ISLET_E_NOT_SUPPORTED = -5;
```

Comment:
Is it possible to rewrite these constants as enum-values?

Say, you define a typedef `islet_rv_t`, and define all these as this enumerated type? Not sure if this is feasible in this C++ code.

Enum-types are helpful as compiler will help checking for returned value. Also, you can see the enum-type-name displayed while debugging.

Reply:
Thanks for suggestion! I added `islet_status_t` enum :)